sssom/validators.py

```diff
@@ -70,6 +71,16 @@ def check_all_prefixes_in_curie_map(msdf: MappingSetDataFrame) -> None:
     :raises ValidationError: If all prefixes not in curie_map.
     """
     prefixes = get_all_prefixes(msdf)
+    prefixes_including_builtins = add_built_in_prefixes_to_prefix_map(msdf.prefix_map)
+    uncommon_maps = {
+        k: v
+        for k, v in msdf.prefix_map.items()
+        if k not in prefixes_including_builtins.keys()
```

Comment:
Is the purpose of this to print out the built_in_prefixes that were previously not in the prefix map, and now were added? If so, shouldn't you loop through prefixes_including_builtins and ehck if `k not in msdf.prefix_map` instead?

Reply:
The purpose here is to check that all prefixes in the `msdf.df` are defined in the `curie_map`, that's all.